src/vscode-bicep/src/visualizer/app/assets/icons/azure/index.ts

```diff
@@ -59,7 +59,83 @@ export async function importResourceIconInline(
       ).default;
     case "microsoft.network/virtualnetworks/subnets":
       return (await import("./custom/subnets.svg")).default;
-
+    case "Microsoft.Cdn/profiles":
```

Comment:
We need to convert the resource types to lowercase.

Reply:
Done and done.